lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -351,3 +359,201 @@ def _load_platform(
 
     platform.initialize()
     return platform
+
+
+@dataclass_json
+@dataclass
+class SigTransformerSchema(schema.Transformer):
```

Comment:
Can you give a minimum values as an example? It should be put in a `"""` format comments, so it can be used to generate document later.

```python
class SigTransformerSchema(schema.Transformer):
    """
    put class comments here
    """
    vhd: ...
```

Reply:
updated